types/wlr_screencopy_v1.c

```diff
@@ -432,6 +467,9 @@ static void capture_output(struct wl_client *wl_client,
 	frame->box = buffer_box;
 	frame->stride = 4 * buffer_box.width; // TODO: depends on read format
 
+	// TODO: Stride is decided by the user for dmabuf, send 0 or something
+	// instead
+
```

Comment:
Need to let the compositor know about the buffer modifiers.

Reply:
> Need to let the compositor know about the buffer modifiers.

Doesn't that happen in `linux-dmabuf-v1-unstable`? I assumed that modifiers would be fully abstracted away at this stage.

Reply:
Actually it does, so its fine.
So its just a matter of changing the wording of the protocol to make stride == 0 "pick whatever".

Reply:
Well, the wording of the protocol is currently "The client **should** then create a buffer with the provided attributes", so it doesn't really _need_ to change, but perhaps we ought to change it for clarification. :)